### PR TITLE
{bio}[foss/2021a] DAS_Tool v1.1.3 w/ Python 3.9.5 + R 4.1.0

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+easyblock = 'Tarball'
+
+name = 'DAS_Tool'
+version = '1.1.1'
```
Comment (boegel) [inline easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:4]: @smoretti Latest is `1.1.5`, let's update to that?

Comment (smoretti) [reply easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:4]: Weird think for that one. It has a small patch file but it should be fine.

The problem is that it downloads a second time the source for R, and the second URL does not exist for 1.1.5 !!!

Comment (boegel) [reply easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:4]: Do you have more details on that?

Comment (smoretti) [reply easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:4]: It does not find the second download link
`ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Couldn't find file DASTool_1.1.5.tar.gz anywhere, and downloading it didn't work either... `

File: easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ('prodigal', '2.6.3'),
+    ('BLAST+', '2.11.0'),
+]
+
+exts_defaultclass = 'RPackage'
+
+exts_list = [
+    (name, version, {
+        'modulename': 'DASTool',
+        'source_tmpl': 'DASTool_%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:35]: @smoretti I guess this should be changed to `%(version)s.tar.gz` too so it works for `1.1.5`?

I'm a bit confused why this works for `1.1.1`...

Comment (smoretti) [reply easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:35]: No, this link (line 35) does not work for version 1.1.5

So maybe we have to stay with version 1.1.1

Comment (smoretti) [reply easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb:35]: Everything looks fine with version 1.1.3

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3135.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/47a7b906b0d0d9acd1441887c887b7a6 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**